README.md

```diff
@@ -70,7 +70,11 @@ The JSON format is:
         * "add"         -> disable add new template menu
         * "add-custom"  -> disable adding custom template
         * "add-default" -> disable adding default template
-        * "add-domain"  -> disable adding additional domains
+        * "custom-settings"  -> disable adding custom settings
+        * "add-domain"       -> disable adding custom domains
+        * "delete-domains"   -> disable deleting all existing custom domains
+        * "add-inputid"      -> disable adding custom input IDs
+        * "delete-inputids"  -> disable deleting all existing custom input IDs
```

Comment:
I think this is a little too granular. 
I think three options should suffice:
`custom-settings` -> Blocks access to the custom settings panel
`custom-domains` -> Can't add/remove any custom domains but leaves custom inputs alone
`custom-input` -> Can't add/remove any custom input ids but leaves custom domains alone